pkgs/top-level/all-packages.nix

```diff
@@ -2509,7 +2509,9 @@ with pkgs;
 
   bukubrow = callPackage ../tools/networking/bukubrow { };
 
-  burpsuite = callPackage ../tools/networking/burpsuite {};
+  burpsuite = callPackage ../tools/networking/burpsuite/community.nix {};
+
+  burpsuite-pro = callPackage ../tools/networking/burpsuite/professional.nix {};
```

Comment:
Can you use a generic.nix style pattern like [all of these packages](https://i.ckie.dev/Zu4IQzD.txt)? Less code duplication.